### ISSUE TITLE
Crash: choco install visualstudio2017-workload-azure -y crashes Powershell at first run

### ISSUE THREAD

**Issue by MaurGi** [jberezanski/ChocolateyPackages#35]
On a Windows 10 fresh VM, we run the following and it kills the powershell instance, even if we use & or Start-Process. 
If we try to run it a second time it works and does not kill the powershell instance - 

For example:
`$process = START-PROCESS powershell -Command {choco install visualstudio2017-workload-azure -y} -PassThru -Wait`

See attached: 
[chocolatey.log](https://github.com/jberezanski/ChocolateyPackages/files/1772866/chocolatey.log)
[choco.summary.log](https://github.com/jberezanski/ChocolateyPackages/files/1772867/choco.summary.log)

Complete setup file:
[dev_machine_setup.zip](https://github.com/jberezanski/ChocolateyPackages/files/1772894/dev_machine_setup.zip)

**Comment by xiw54** [comment]
Another example which didn't work:
$process = START-PROCESS powershell $file_path -PassThru -Wait
It calls another ps script executes one-line command "choco install visualstudio2017-workload-azure -y"

**Comment by jberezanski** [comment]
I have a hypothesis, but I'll need the Visual Studio logs to confirm it: `$Env:Temp\chocolatey\dd_*.log`

(The Azure PowerShell installer had a nasty habit of killing powershell.exe processes - Azure/azure-powershell#2130. I also found this: https://github.com/MicrosoftDocs/visualstudio-docs/issues/301)

**Comment by MaurGi** [comment]
Xin – can you load the logs on the GitHub issue please?

Thanks!
-MG-

**Comment by xiw54** [comment]
Thank you Jakub, please see the attached logs.

Thank you!

Xin
[logs-a.zip](https://github.com/jberezanski/ChocolateyPackages/files/1773322/logs-a.zip)
[logs-b.zip](https://github.com/jberezanski/ChocolateyPackages/files/1773323/logs-b.zip)
[logs-c.zip](https://github.com/jberezanski/ChocolateyPackages/files/1773324/logs-c.zip)

**Comment by jberezanski** [comment]
Yes, I believe I was right.

In dd_setup_20180301211242_002_Microsoft.Azure.PowerShell.log:

````
MSI (s) (C4!CC) [21:13:16:602]: PROPERTY CHANGE: Adding WixCloseApplicationsDeferred property. Its value is 'powershell.exe7150000powershell_ise.exe7150000'.
MSI (s) (C4!CC) [21:13:16:602]: Doing action: WixCloseApplicationsDeferred
````
This is the Azure PowerShell installer killing all powershell.exe and powershell_ise.exe processes. So it wasn't fixed after all, at least not in version 5.0.0 installed by Visual Studio. 😞 

In the workstation setup script at my company we work around it by renaming the PowerShell executable (actually, copying it under a new name, such as posh.exe) and using it to run the relevant portion of the workstation config process.

**Comment by xiw54** [comment]
Thanks a lot for figuring this out! That’s a nice workaround. I’ll try it :)
